src/com/projectkorra/projectkorra/command/HelpCommand.java

```diff
@@ -153,23 +156,20 @@ public void execute(final CommandSender sender, final List<String> args) {
-			sender.sendMessage(ChatColor.YELLOW + this.learnMore + ChatColor.DARK_AQUA + "http://projectkorra.com/");
+			sender.sendMessage(Element.WATER.getColor() + this.elementHelp.get(Element.WATER).replaceAll("(?i)/b display water", Element.WATER.getSubColor() + "/b display water" + Element.WATER.getColor()));
 		} else if (Arrays.asList(Commands.earthaliases).contains(arg)) {
-			sender.sendMessage(Element.EARTH.getColor() + this.earth.replace("/b display Earth", Element.EARTH.getSubColor() + "/b display Earth" + Element.EARTH.getColor()));
-			sender.sendMessage(ChatColor.YELLOW + this.learnMore + ChatColor.DARK_AQUA + "http://projectkorra.com/");
+			sender.sendMessage(Element.EARTH.getColor() + this.elementHelp.get(Element.EARTH).replaceAll("(?i)/b display earth", Element.EARTH.getSubColor() + "/b display earth" + Element.EARTH.getColor()));
 		} else if (Arrays.asList(Commands.firealiases).contains(arg)) {
-			sender.sendMessage(Element.FIRE.getColor() + this.fire.replace("/b display Fire", Element.FIRE.getSubColor() + "/b display Fire" + Element.FIRE.getColor()));
-			sender.sendMessage(ChatColor.YELLOW + this.learnMore + ChatColor.DARK_AQUA + "http://projectkorra.com/");
+			sender.sendMessage(Element.FIRE.getColor() + this.elementHelp.get(Element.FIRE).replaceAll("(?i)/b display fire", Element.FIRE.getSubColor() + "/b display fire" + Element.FIRE.getColor()));
 		} else if (Arrays.asList(Commands.chialiases).contains(arg)) {
-			sender.sendMessage(Element.CHI.getColor() + this.chi.replace("/b display Chi", Element.CHI.getSubColor() + "/b display Chi" + Element.CHI.getColor()));
-			sender.sendMessage(ChatColor.YELLOW + this.learnMore + ChatColor.DARK_AQUA + "http://projectkorra.com/");
+			sender.sendMessage(Element.CHI.getColor() + this.elementHelp.get(Element.CHI).replaceAll("(?i)/b display chi", Element.CHI.getSubColor() + "/b display chi" + Element.CHI.getColor()));
 		} else if (Arrays.asList(Commands.avataraliases).contains(arg)) {
-			sender.sendMessage(Element.AVATAR.getColor() + this.avatar.replace("/b display Avatar", Element.AVATAR.getSubColor() + "/b display Avatar" + Element.AVATAR.getColor()));
-			sender.sendMessage(ChatColor.YELLOW + this.learnMore + ChatColor.DARK_AQUA + "http://projectkorra.com/");
+			sender.sendMessage(Element.AVATAR.getColor() + this.elementHelp.get(Element.AVATAR).replaceAll("(?i)/b display avatar", Element.AVATAR.getSubColor() + "/b display avatar" + Element.AVATAR.getColor()));
+		} else if (Element.getElement(arg) != null && elementHelp.containsKey(Element.getElement(arg))) {
+			Element element = Element.getElement(arg);
+			sender.sendMessage(element.getColor() + this.elementHelp.get(element).replaceAll("(?i)/b display " + element.getName().toLowerCase(), element.getSubColor() + "/b display " + element.getName().toLowerCase() + element.getColor()));
```

Comment:
Do we really need to differentiate different elements now? It seems like we could just use the code in the last else if.

Reply:
Ah no, we want to support the hardcoded core elemental aliases and custom elements don't have those. Nevermind 🤪 That being said, we may want to rethink that in the future.

Reply:
Yeah, I also removed it at first before realizing lol. We will indeed need to review in future